client/layout/masterbar/logged-in.jsx

```diff
@@ -17,6 +17,9 @@ import config from 'config';
 import { preload } from 'sections-preload';
 import ResumeEditing from 'my-sites/resume-editing';
 import { setNextLayoutFocus } from 'state/ui/layout-focus/actions';
+import { decorate } from 'plugins/helpers';
+
+const preloadStats = preload.bind( null, 'stats' );
```

Comment:
Given that it's such a trivial refactor, I don't care whether it goes in this PR or another — I just wanted to shut eslint up. :)